tests/devices/qutrit_mixed/test_qutrit_mixed_apply_operation.py

```diff
@@ -0,0 +1,484 @@
+        op._batch_size = None  # pylint:disable=protected-access
+        state = method(op, state)
+        assert state.shape == (3, 3, 3)
+        assert op.batch_size == self.num_batched
+
+
+@pytest.mark.parametrize("ml_framework", ml_frameworks_list)
+@pytest.mark.parametrize("method", methods)
+class TestChannels:  # pylint: disable=too-few-public-methods
+    """Tests that Channel operations are applied correctly."""
+
+    class CustomChannel(Channel):
+        num_params = 1
+        num_wires = 1
+
+        def __init__(self, p, wires, id=None):
+            super().__init__(p, wires=wires, id=id)
+
+        @staticmethod
+        def compute_kraus_matrices(p):
+            K0 = (np.sqrt(1 - p) * math.cast_like(np.eye(3), p)).astype(complex)
+            K1 = (
+                np.sqrt(p) * math.cast_like(np.array([[0, 0, 1], [1, 0, 0], [0, 1, 0]]), p)
+            ).astype(complex)
+            return [K0, K1]
+
+    def test_non_broadcasted_state(self, method, ml_framework):
+        """Tests that Channel operations are applied correctly to a state."""
+        state = get_random_mixed_state(2)
+        test_channel = self.CustomChannel(0.3, wires=1)
+        res = method(test_channel, math.asarray(state, like=ml_framework))
+        flattened_state = state.reshape(9, 9)
+
+        mat = test_channel.kraus_matrices()
+
+        expanded_mats = [np.kron(np.eye(3), mat[i]) for i in range(len(mat))]
+        expected = np.zeros((9, 9)).astype(complex)
+        for i in range(len(mat)):
+            expanded_mat = expanded_mats[i]
+            adjoint_mat = np.conj(expanded_mat).T
+            expected += expanded_mat @ flattened_state @ adjoint_mat
+        expected = expected.reshape([3] * 4)
+
+        assert qml.math.get_interface(res) == ml_framework
+        assert qml.math.allclose(res, expected)
+
+    def test_broadcasted_state(self, method, ml_framework):
+        """Tests that Channel operations are applied correctly to a batched state."""
+        if method is apply_operation_tensordot:
+            pytest.skip("Tensordot doesn't support batched operations.")
+        state = [get_random_mixed_state(2) for _ in range(3)]
+        test_channel = self.CustomChannel(0.3, wires=1)
+        res = method(test_channel, math.asarray(state, like=ml_framework))
+
+        mat = test_channel.kraus_matrices()
+        expanded_mats = [np.kron(np.eye(3), mat[i]) for i in range(len(mat))]
+        expected = [np.zeros((9, 9)).astype(complex) for _ in range(3)]
+        for i in range(3):
+            flattened_state = state[i].reshape(9, 9)
+            for j in range(len(mat)):
+                expanded_mat = expanded_mats[j]
+                adjoint_mat = np.conj(expanded_mat).T
+                expected[i] += expanded_mat @ flattened_state @ adjoint_mat
+            expected[i] = expected[i].reshape([3] * 4)
+
+        assert qml.math.get_interface(res) == ml_framework
+        assert qml.math.allclose(res, expected)
```

Comment:
I plan to add tests for Grad/Jacobian when there is measurement functionality this also helps break up this PR.